test_opensearchpy/test_async/test_connection.py

```diff
@@ -97,6 +97,12 @@ async def test_ssl_context(self) -> None:
         assert con.use_ssl
         assert con.session.connector._ssl == context
 
+    async def test_ssl_assert_hostname(self) -> None:
```

Comment:
Add a test for the default value and a True value, too.

Reply:
Done.